src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -824,6 +824,8 @@ settings.set( "lua.autocomplete", true )
 settings.set( "list.show_hidden", false )
 if term.isColour() then
     settings.set( "bios.use_multishell", true )
+    settings.set( "list.file_color", "white" )
+    settings.set( "list.directory_color", "green" )
```

Comment:
Haven't tested it, but it doesn't look like this would work. Wouldn't this make it white & green every time, regardless of what the user sets?